recipes/palmari/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "palmari" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/palmari-{{ version }}.tar.gz
+  sha256: 6c7b4bd481ce6d83e944010bc4495f2a6c2e5137fcbafc7a04d81a0de8c93888
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - dask-core ==2022.1.0
+    - dask-image ==2021.12.0
```

Comment:
@hippover, do you think we can be a bit more flexible here? Do newer dask versions work or was this the last one to work?